app/controllers/courses_controller.rb

```diff
@@ -237,6 +245,34 @@ def wiki_params
       .permit(:language, :project)
   end
 
+  def courses_wikis_params
+    params
+      .require(:course)
+      .permit(wikis: [:language, :project])
+  end
+
+  def update_courses_wikis
```

Comment:
I think this method can be simplified a lot. We can lean on Rails for this kind of thing, because the CoursesWikis table represents a "has many, through" relationship. Ideally, we shouldn't need to get to the level ids.

For example, `course.wikis.delete(wiki)` and `course.wikis.push(wiki)` will add or remove a CoursesWikis record for provided `wiki`.